src/main/java/org/sonar/plugins/checkstyle/metadata/CheckstyleMetadata.java

```diff
@@ -239,30 +304,38 @@ public static boolean isSetter(Method method) {
      * It converts the name received from ModuleDetails to Sonar rule name format
      * e.g. RightCurlyCheck -> Right Curly Check
      *
-     * @param name the name fetched from ModuleDetails
+     * @param checkName the name fetched from ModuleDetails
      * @return modifiedName
      */
-    private static String convertName(String name) {
+    private static String convertName(String checkName) {
         final int capacity = 1024;
         final StringBuilder result = new StringBuilder(capacity);
-        for (int i = 0; i < name.length(); i++) {
-            result.append(name.charAt(i));
-            if (i + 1 < name.length() && Character.isUpperCase(name.charAt(i + 1))) {
+        for (int i = 0; i < checkName.length(); i++) {
+            result.append(checkName.charAt(i));
+            if (i + 1 < checkName.length() && Character.isUpperCase(checkName.charAt(i + 1))) {
                 result.append(' ');
             }
         }
         return result.toString();
     }
 
+    /**
+     * Create internal key composed of parents of module.
+     *
+     * @param moduleDetails module metadata
+     * @return internalKey
+     */
     private static String convertInternalKey(ModuleDetails moduleDetails) {
```

Comment:
all method that convert to smth shoud have in name `ToSomthing`

make name like: `getInternalKey`
and make it public to reuse in XxxxTest.java

Reply:
Done in new reorganized PR https://github.com/checkstyle/sonar-checkstyle/pull/306